File: lib/src/model/FortuneUser/components/user_manifest.dart
import 'package:cloud_firestore/cloud_firestore.dart';

class UserManifest {
  String manifestText;
  Timestamp manifestDate;
  String manifestNo;

  UserManifest({
    required this.manifestText,
    required this.manifestDate,
    required this.manifestNo,
  });

  factory UserManifest.fromJson(Map<String, dynamic> json) => UserManifest(
        manifestText: json["manifestText"],
        manifestDate: json["manifestDate"] as Timestamp,
        manifestNo: json["manifestNo"],
      );

  Map<String, dynamic> toJson() => {
        "manifestText": manifestText,
        "manifestDate": manifestDate,
        "manifestNo": manifestNo,
      };
}

File: functions/model/FortuneUser/components/UserManifest.js
class UserManifest {
    constructor({ manifestText, manifestDate, manifestNo }) {

        this.manifestText = manifestText,
            this.manifestDate = manifestDate,
            this.manifestNo = manifestNo
    }

    static fromJson(json) {
        return new UserManifest({
            manifestText: json["manifestText"],
            manifestDate: json["manifestDate"],
            manifestNo: json["manifestNo"],


        });
    }

    toJson() {
        return {
            "manifestText": this.manifestText,
            "manifestDate": this.manifestDate,
            "manifestNo": this.manifestNo,
        };
    }
}
module.exports = { UserManifest };


